test/conformance/kuttl/validate/clusterpolicy/standard/enforce/failure-policy-ignore-anchor/README.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+This test verifies that when failurePolicy is set to to Ignore for a policy that was set to Enforce, Admission webhook denies requests when validation of a resource fails. The error should not get consumed by ignore failurePolicy 
```

Comment:
It is expected to bypass the fail/error results when applying a validate rule in enforce mode if the failure policy is Ignore. Why changing it?

Reply:
We only bypass errors when failure policy is set to ignore

Reply:
https://github.com/kyverno/kyverno/blob/main/pkg/utils/engine/response.go

Reply:
In the case in the proof manifest, a fail case is wrongly categorised is error, this causes the policy to be bypassed when it should have blocked.

Reply:
> The failure policy of a webhook controls what happens when a request either takes too long to complete or encounters a failure. The default of Fail indicates the request will fail. A value of Ignore means the request will be allowed regardless of its status.

You are right, Ignore bypasses request when there's an error applying the policy, say a timeout.